src/js/reusable-charts/horizontal-bar-chart.js

```diff
@@ -312,6 +320,12 @@ export function horizontalBarChart() {
         return exportData;
     }
 
+    function _maxX() {
```

Comment:
not sure this needed, right? or line 80 is unecessary.

Reply:
i think line 80 is unnecessary

Reply:
_maxX is necessary because data may change in which case you will need to re-calculate maxX. An alternative would be to update it in chart.data